pymysqlreplication/tests/test_event.py

```diff
@@ -11,25 +12,20 @@ def tearDown(self):
         self.execute("SET SESSION binlog_rows_query_log_events=0")
         super(BinLogEventTestCase, self).tearDown()
 
-    target_fields = [
-        "timestamp",
-        "log_pos",
-        "event_size",
-        "read_bytes"
-    ]
+    target_fields = ["timestamp", "log_pos", "event_size", "read_bytes"]
```

Comment:
Is there a reason you put the target_field out of scope?

Reply:
i just thought it can be reused when write other test case.